docs/networking.md

```diff
@@ -350,6 +351,77 @@ $ kops create cluster \
 
 In case of any issues the directory `/var/log/aws-routed-eni` contains the log files of the CNI plugin. This directory is located in all the nodes in the cluster.
 
+### Cilium Example for CNI and Network Policy
+
+Cilium is open source software for transparently securing the network connectivity between application services deployed using Linux container management platforms like Docker and Kubernetes.
+
+#### Installing Cilium on a new Cluster
+
+The following command sets up a cluster, in HA mode, with Cilium as the CNI and networking policy provider
+
+```console
+$ export ZONES=mylistofzones
+$ kops create cluster \
+  --zones $ZONES \
+  --master-zones $ZONES \
+  --master-size m4.large \
+  --node-size m4.large \
+  --networking cilium\
+  --yes \
+  --name myclustername.mydns.io
+```
+
+The above will deploy a daemonset installation which requires K8s 1.7.x or above.
+
+#### Configuring Cilium
+
+The following command registers a cluster, but doesn't create it yet
+
+```console
+$ export ZONES=mylistofzones
+$ kops create cluster \
+  --zones $ZONES \
+  --master-zones $ZONES \
+  --master-size m4.large \
+  --node-size m4.large \
+  --networking cilium\
+  --name myclustername.mydns.io
+```
+
+`kops edit cluster`  will show you a block like this:
+
+```
+  networking:
+    cilium: {}
+```
+
+You can adjust Cilium agent configuration with most options that are available in [cilium-agent command reference](http://cilium.readthedocs.io/en/v1.0/cmdref/cilium-agent/).
+
+E.g enabling logstash integration would require you to change above block to
+
+```
+  networking:
+    cilium:
+      logstash: true
+```
+
+The following command will create your cluster with desired Cilium configuration
+
+```console
+$ kops update cluster myclustername.mydns.io --yes
+```
+
+#### Getting help with Cilium
+
+For problems with deploying Canal please post an issue to Github:
```

Comment:
I think you meant "For problems with deploying Cilium..." ;-)

Reply:
Whoops!